app/main/views/frameworks.py

```diff
@@ -340,13 +339,14 @@ def framework_submission_services(framework_slug, lot_slug):
                     framework_slug=framework_slug, lot_slug=lot_slug, service_id=draft['id'])
         )
 
+    lot_service_sections = content_loader.get_manifest(
+        framework_slug,
+        'edit_submission',
+    ).filter(context={'lot': lot_slug}, inplace_allowed=True)
+
     with logged_duration(message="Annotated draft details in {duration_real}s"):
-        for draft in chain(drafts, complete_drafts):
-            draft['priceString'] = format_service_price(draft)
-            sections = content_loader.get_manifest(
-                framework_slug,
-                'edit_submission',
-            ).filter(draft, inplace_allowed=True).summary(draft, inplace_allowed=True)
+        for draft in drafts:
+            sections = lot_service_sections.summary(draft, inplace_allowed=True)
```

Comment:
I'm a _little bit_ nervous about this change, especially used in conjunction with `inplace_allowed`. I didn't make this optimization because we're now mutating the same `ContentManifest` over and over again, overwriting it with each new service's details. And with the obfuscated design of the content loader I couldn't be certain that no data leaks from one iteration to the next.